srcpkgs/StormLib/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'StormLib'
```

Comment:
The new packages should have their own commits.

Reply:
I forgot that.

Reply:
I opened PR for [Stormlib](34108)

Reply:
No, the commit should still be in this PR, just needs to be a separate commit.

Reply:
It would not be better for separate PR, I say StormLib, smpq and devilutionX, each having your PR, devilutionX PR would only update, other PR would be new packages. Because it is easier to find failures, to compile everything at once and have strange results.

Reply:
If those packages are required for the update, this should be in this PR.

Reply:
The creator of devilutionX software, says that the smpq package is optional, but it is not, needs this package to compile. 

https://github.com/diasurgical/devilutionX/blob/master/docs/building.md#if-you-want-to-build-the-devilutionxmpq-file-optional

Reply:
This matter isn't resolved. Separate these new packages into individual commits in this PR.

Reply:
Done!